openexp/_canvas/psycho.py

```diff
@@ -45,6 +45,18 @@
 	raise osexception(
 		u'PsychoPy is missing the GratingStim() class. Please update your version of PsychoPy! For installation instructions, please visit http://www.psychopy.org/.')
 
+import psychopy, warnings
+from distutils.version import StrictVersion
+try:
+    PsyPy_vers = StrictVersion(psychopy.__version__)
```

Comment:
You're using `CamelCase`, whereas OpenSesame style is to use `lowercase_with_underscores_if_necessary`.